source/_patterns/colors/_intro.demonstration.scss

```diff
@@ -0,0 +1,18 @@
+main {
+	figure {
+		margin: 0;
+		display: grid;
+		grid-auto-columns: 50% 50%;
+		grid: 1fr 1fr;
+		grid-gap: 1rem;
+		grid-auto-flow: column;
+	}
+
+	img {
+		width: 100%;
+		max-width: 100%;
+		border-radius: 0.875rem;
+		box-shadow: rgb(60 64 67 / 30%) 0px 1px 2px 0px,
+			rgb(60 64 67 / 15%) 0px 1px 3px 1px;
+	}
```

Comment:
that's quite a global statement, have a look at e.g. https://db-ui.github.io/base/review/feat-colors-docs/?p=pages-intro

Reply:
yes, changed that locally but forgot to push yesterday ;)